cmd/maintainer.go

```diff
@@ -61,7 +61,24 @@ func maintainers(cmd *cobra.Command, args []string) error {
 		)
 	}
 
-	maintainer.Initialize(ctx, clientConfig.Maintainer, btcChain, chain)
+	_, tbtcChain, _, _, _, err := ethereum.Connect(
+		ctx,
+		clientConfig.Ethereum,
+	)
+	if err != nil {
+		return fmt.Errorf(
+			"could not connect to Bitcoin difficulty chain: [%v]",
```

Comment:
This is not Bitcoin difficulty chain but the tBTC chain.

Reply:
4ee460edecc90e93e3047c04dfd5a9f26f84db3c